src/a4/domain/OpenSpace.java

```diff
@@ -1,5 +1,16 @@
 package a4.domain;
 
-public class OpenSpace {
+public class OpenSpace extends BoardSpace {
+	public OpenSpace() {
+	}
 
```

Comment:
I am ok with however we want to do it! We can put name in BoardSpace? I just don't see the name for Open space getting set anywhere in the factory so I am wondering where we give the space a name? If we want to move it that's totally fine though!